navigation/ca_hector/launch/geotiff_mapper.launch

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+
+<launch>
+
+  <arg name="trajectory_source_frame_name" default="/base_link"/>
```

Comment:
A few considerations here:

- Remove the arguments and use directly the parameters inside the node.
- Use base_footprint instead of base_link
- Consider using YAML files to load the parameters